lib/determinator.rb

```diff
@@ -15,11 +15,20 @@ def self.configure(retrieval:, errors: nil, missing_feature: nil)
     @instance = Control.new(retrieval: retrieval)
   end
 
+  # @return [Determinator::Control] The currently active instance of determinator.
+  # @raises [RuntimeError] If no Determinator instance is set up (with `.configure`)
   def self.instance
     raise "No singleton Determinator instance defined" unless @instance
     @instance
   end
 
+  # Returns the feature with the given name as Determinator uses it. This is useful for
+  # debugging issues with the retrieval mechanism which delivers features to Determinator.
+  # @returns [Determinator::Feature,nil] The feature details Determinator would use for a determination right now.
+  def self.feature_details(name)
```

Comment:
why not just `feature(name)`?

Reply:
I thought it wasn't quite descriptive enough (is `feature` a verb or a noun — we have `configure` on there, and `instance`; so we have one example of each 🤦‍♂️ ) which might leave people thinking it's something that _does_ something to a feature. Happy to change it back though if it doesn't add enough clarity!